docs/testplan/HA-SmartSwitch-test-plan.md

```diff
@@ -0,0 +1,188 @@
+
+For HA testbeds, the 2 smartswitch DUTs will share the same set of cEOS dockers as the neighbors, and same PTF docker for packets injecting and sniffing. 
+
+![physical-connection](./Img/topo_smartswitch_ha.png)
+
+Please refer to [sonic-mgmt testbed overview](https://github.com/sonic-net/sonic-mgmt/blob/master/docs/testbed/README.testbed.Overview.md) for more about testbed setup and diagrams, also [HA Topology](https://github.com/sonic-net/SONiC/blob/master/doc/smart-switch/high-availability/smart-switch-ha-hld.md#4-network-physical-topology) for HA Network Physical Design. 
+
+### Setup configuration  
+
+The production scenario simulated with this testbed, is a VM-to-VM traffic scenario. Basically 2 VMs located in perhaps two different clusters in the data center, try to communicate with each other. Traffic passes through the HA set under test. Assuming dpu0 in SmartSwitch0 will be the Active node and dpu0 in SmartSwitch1 shall be set to Standby. Both DPUs will share same network configurations.
+
+In case the traffic lands on standby node, it will be tunnelled through T0 neighbor, to the active node, and eventually sent out to destination VM. Diagram below shows the logical path of the traffic. 
+![vm-vm-standby](./Img/ssw-ha-test-plan-logic-standby.png)
+
+In case the traffic lands on active node, the path will be like below. 
+![vm-vm-active](./Img/ssw-ha-testplan-logic-active.png)
+
+
+## Test Methodology
+
+1. Test scenario is VM-to-VM traffic (we may add other critical scenarios, such as private link, in the future). 
+1. Open flow rules are configured on OVS Bridges to control the traffic flow:
+    * All packets sent out from Neighbor are only forwarded to DUT
+    * All packets sent out from DUT are forwarded to Neighbor and PTF
+    * All packets sent out from PTF are only forwarded to DUT
+1. Traffic will be sent from PTF docker, and sniffed on PTF docker by sonic-mgmt test scripts.
+1. Depending on the test cases, switchovers will be triggered and verified. 
+1. Control plane status, DPU counters, metering data, and flow table diff between active/standby will be measured for the tests.
+
+## Test Plan
+Assuming there is a pair of DPUs in the system, and at the step of test setup, we have DPU-1 as active, DPU-2 as standby. 
+
+Traffic will be injected by PTF through the T2 neighbors' OVS bridges. PTF will also sniff on the interfaces binding to the T2 neighbors.  
+
+For some of the cases, you will see two versions, one is traffic sending through DPU1, and the other one is traffic sending through DPU2. The purpose of  it is to verify:
+* Traffic shouldn’t be disrupted when failures happen on standby side.
+* Traffic should be tunneled if landing on standby side. 
+
+The name convention of a test case will be “\<Test Scenario\>-[Active|Standby]”, indicating the traffic is sent through the initial active or standby side. 
+
+### Module 1 Normal OP 
```

Comment:
it might be better to call this module "Baseline" or  "Steady State" or something similar.

Reply:
Changed to "Steady State"